src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -634,18 +634,6 @@ private static bool SupressesImport(MemberInfo member, Dictionary<MethodInfo, At
             return false;
         }
 
-        private bool AllowsRankReduction(MethodInfo method)
```

Comment:
I am a little confused about that this method now can be deleted. Was it not used before?

Reply:
Yes, this method is unused so I'm deleting it.